generator/internal/dart/darttemplate.go

```diff
@@ -25,7 +25,10 @@ import (
 )
 
 type modelAnnotations struct {
-	PackageName       string
+	// The Dart package name (e.g. google_cloud_secretmanager).
+	PackageName string
+	// Name of the API in snake_format (e.g. secretmanager).
+	NameToSnake       string
```

Comment:
I have been trying to move away from `NameToSnake` and `NameToPascal` and trying to say the name of what it is, maybe in this case `MainFileName` ?

Reply:
sounds good; renamed